src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs

```diff
@@ -131,13 +139,20 @@ internal class HttpSource
 
                 using (var response = await _client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead))
                 {
+                    if (!throwNotFound && response.StatusCode == HttpStatusCode.NotFound)
```

Comment:
Where is the change to change the URL from v3 to v2 in the beta5 branch?

-----Original Message-----
From: "Wei Wang" <notifications@github.com>
Sent: ‎6/‎12/‎2015 5:11 PM
To: "aspnet/dnx" <dnx@noreply.github.com>
Cc: "Yishai Galatzer" <yishai_galatzer@yahoo.com>
Subject: Re: [dnx] Add NuGet v3 feed support (#2058)

In src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs:
> @@ -131,13 +139,20 @@ internal class HttpSource
>  
>                  using (var response = await _client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead))
>                  {
> +                    if (!throwNotFound && response.StatusCode == HttpStatusCode.NotFound)
For v3 feed, FindPackagesById() becomes construct the URI by ourselves and check whether the URI points to a JSON. A 404 page means the packages is not on this feed.
—
Reply to this email directly or view it on GitHub.